release_bot/github.py

```diff
@@ -429,6 +429,7 @@ def make_release_pr(self, new_pr):
             name, email = self.get_user_contact()
             repo.set_credentials(name, email)
             repo.set_credential_store()
+            repo.checkout('master')
```

Comment:
Could you please elaborate why is this needed? Two lines below there is another checkout. Ideally please include a comment in the code.